UnitystationLauncher/ViewModels/SecurityPanelViewModel.cs

```diff
@@ -0,0 +1,538 @@
+
+                    string response = await msgBox.Show();
+
+                    await _writer.WriteLineAsync(response == "No" ? false.ToString() : true.ToString());
+                    await _writer.FlushAsync();
+                    return Task.CompletedTask;
+                });
+            }
+            else if (ClientRequest.API_URL.ToString() == requests[0])
+            {
+                RxApp.MainThreadScheduler.ScheduleAsync(async (_, _) =>
+                {
+                    IMsBoxWindow<string> msgBox = MessageBoxBuilder.CreateMessageBox(
+                        MessageBoxButtons.YesNo,
+                        string.Empty,
+                        $"The build would like to send an API request to, {requests[1]} " + @"
+do you allow this fork to now on access this domain
+Justification given by the Fork : " + requests[2]);
+
+
+                    string response = await msgBox.Show();
+
+                    await _writer.WriteLineAsync(response == "No" ? false.ToString() : true.ToString());
+                    await _writer.FlushAsync();
+                    return Task.CompletedTask;
+                });
+            }
+            else if (ClientRequest.Host_Trust_Mode.ToString() == requests[0])
+            {
+                RxApp.MainThreadScheduler.ScheduleAsync(async (_, _) =>
+                {
+                    IMsBoxWindow<string> msgBox = MessageBoxBuilder.CreateMessageBox(
+                        MessageBoxButtons.YesNo,
+                        string.Empty,
+                        @" Trusted mode automatically allows every API and open URL action to happen without prompt, this also enables the 
+Variable viewer ( Application that can modify the games Data ) that Could potentially be used to Perform malicious actions on your PC,
+ The main purpose of this Prompt is to allow the Variable viewer (Variable editing), 
+What follows is given by the build, we do not control what is written in the Following text So treat with caution and use your brain
+ Justification : " + requests[1]); //TODO Add text
+
+                    string response = await msgBox.Show();
+
+                    await _writer.WriteLineAsync(response == "No" ? false.ToString() : true.ToString());
+                    await _writer.FlushAsync();
+                    return Task.CompletedTask;
+                });
+            }
+        }
+    }
+
+    public void Dispose()
+    {
+        _serverPipe.Dispose();
+        _reader?.Dispose();
+        _writer?.Dispose();
+    }
+}
+
+public class SecurityPanelViewModel : PanelBase
+{
+    public override string Name => "Security";
+    public override bool IsEnabled => false;
+
+    private readonly IAssemblyChecker _IAssemblyChecker;
+
+    public SecurityPanelViewModel(IAssemblyChecker assemblyChecker)
+    {
+        _IAssemblyChecker = assemblyChecker;
+    }
+
+
+    public override void Refresh()
+    {
+        //what does this do?!?
```

Comment:
This is for manually refreshing the UI if needed, rather than using observables which have a memory leak in Avalonia. If there isn't anything to update, its fine to leave this empty.

Reply:
was codisy complaining about empty function